src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.SslProtocols.cs

```diff
@@ -91,6 +91,12 @@ public void DisabledProtocols_SetSslProtocols_ThrowsException(SslProtocols disab
         [InlineData(SslProtocols.Tls12, true)]
         public async Task GetAsync_AllowedSSLVersion_Succeeds(SslProtocols acceptedProtocol, bool requestOnlyThisProtocol)
         {
+            if (ManagedHandlerTestHelpers.IsEnabled)
+            {
+                // TODO #21452: The managed handler is failing.
+                return;
+            }
```

Comment:
Note, too, that it's just using SslStream to establish the connection to the server.  If there's a failure here, it's due to SslStream, and it's something that should be looked into.  We've now caught several general SslStream bugs due to HttpClient now using it via ManagedHandler, highlighting both bugs and large test gaps in System.Net.Security.